common/static/common/js/discussion/views/discussion_inline_view.js

```diff
@@ -164,9 +167,8 @@
                 this.hideDiscussion();
             } else {
                 this.toggleDiscussionBtn.addClass('shown');
-                this.toggleDiscussionBtn.find('.button-text').html(gettext('Hide Discussion'));
+                edx.HtmlUtils.setHtml(this.toggleDiscussionBtn.find('.button-text'), gettext('Hide Discussion'));
```

Comment:
Nit: this would be better left as it was, but just using ``.text`` instead of ``.html`` to ensure that it is escaped.